config/config.go

```diff
@@ -45,21 +45,21 @@ type Config struct {
 
 func FromFlags() (*Config, error) {
 	cfg := &Config{}
-	var evmNetwork, coinbase, gas, coa, key, keysPath string
+	var evmNetwork, coinbase, gas, coa, key, keysPath, flowChainID string
 
 	// parse from flags
-	flag.StringVar(&cfg.DatabaseDir, "database-dir", "./db", "path to the directory for the database")
-	flag.StringVar(&cfg.RPCHost, "rpc-host", "", "host for the RPC API server")
-	flag.IntVar(&cfg.RPCPort, "rpc-port", 8545, "port for the RPC API server")
-	flag.StringVar(&cfg.AccessNodeGRPCHost, "access-node-grpc-host", "localhost:3569", "host to the flow access node gRPC API")
+	flag.StringVar(&cfg.DatabaseDir, "database-dir", "./db", "Path to the directory for the database")
+	flag.StringVar(&cfg.RPCHost, "rpc-host", "", "Host for the RPC API server")
+	flag.IntVar(&cfg.RPCPort, "rpc-port", 8545, "Port for the RPC API server")
+	flag.StringVar(&cfg.AccessNodeGRPCHost, "access-node-grpc-host", "localhost:3569", "Host to the flow access node gRPC API")
 	flag.StringVar(&evmNetwork, "evm-network-id", "testnet", "EVM network ID (testnet, mainnet)")
-	flag.StringVar(&cfg.FlowNetworkID, "flow-network-id", "emulator", "EVM network ID (emulator, previewnet)")
-	flag.StringVar(&coinbase, "coinbase", "", "coinbase address to use for fee collection")
-	flag.StringVar(&gas, "gas-price", "1", "static gas price used for EVM transactions")
+	flag.StringVar(&flowChainID, "flow-network-id", "flow-emulator", "Flow network ID (flow-emulator, flow-previewnet)")
+	flag.StringVar(&coinbase, "coinbase", "", "Coinbase address to use for fee collection")
```

Comment:
A bit out of context of this PR, but it occurred to me that we might better fetch this from the `COA` resource that resides in the `coa-address`. We can do this either through a script on startup and memoize the value.

Reply:
ohh you are right, can you open an issue for this.

Reply:
Sure thing :+1:

Reply:
Opened https://github.com/onflow/flow-evm-gateway/issues/122